pkg/utils/utils.go

```diff
@@ -1195,3 +1195,36 @@ func CompareVersion(currentVersion string, minVersion string) int {
 	}
 	return current.Compare(minimum)
 }
+
+func RetrieveVddkLogLevel(params map[string]interface{}, config *rest.Config, logger logrus.FieldLogger) error {
+	var err error // Declare here to avoid shadowing on using in cluster config only
+	if config == nil {
+		config, err = GetKubeClientConfig()
+		if err != nil {
+			logger.WithError(err).Errorf("Failed to get k8s inClusterConfig")
+			return errors.Wrap(err, "Could not retrieve in-cluster config")
+		}
+	}
+	kubeClient, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		logger.WithError(err).Errorf("Failed to get k8s clientset from the given config: %v", config)
+		return err
+	}
+	veleroNs, exist := os.LookupEnv("VELERO_NAMESPACE")
+	if !exist {
+		errMsg := "Failed to lookup the ENV variable for velero namespace"
+		logger.Error(errMsg)
+		return errors.New(errMsg)
+	}
+	vddkLogLevelConfigMap, err := kubeClient.CoreV1().ConfigMaps(veleroNs).Get(context.TODO(), constants.VddkLogLevel, metav1.GetOptions{})
```

Comment:
It is not right and not extensible to name the config as `vddk-log-level`.  Please use something like `vddk-config`.